tarodb/asset_minting.go

```diff
@@ -535,8 +535,8 @@ func (a *AssetMintingStore) FetchNonFinalBatches(
 				}
 				batches[i].GenesisPacket = &tarogarden.FundedPsbt{
 					Pkt: genesisPkt,
-					ChangeOutputIndex: extractSqlInt16[uint32](
-						batch.MintingOutputIndex,
+					ChangeOutputIndex: extractSqlInt32[int32](
```

Comment:
`sqlInt16` and `extractSqlInt16` are now unused and blocking CI lint checks. Could make this handling more generic if we expect to use int16 later?

Reply:
Removed them for now. I think most of our use cases should be either int32 (anything chain index related) or int8 (enums and things like that).